tests/testthat/test-quo.R

```diff
@@ -1,25 +1,81 @@
+# quo_c ----
+## Test 1: `quo_c` works in concatenating and indexing quosures ----
+test_that("quo_c Test 1: `quo_c` works in concatenating and indexing quosures", {
+  x <- quo(USUBJID)
+  y <- quo(STUDYID)
+
+  expect_equal(
+    expected = quo(USUBJID),
+    object = quo_c(x, NULL, y)[[1]]
+  )
+  expect_equal(
+    expected = quo(STUDYID),
+    object = quo_c(x, NULL, y)[[2]]
+  )
+})
+
+## Test 2: `quo_c` returns error if non-quosures are input ----
+test_that("quo_c Test 2: `quo_c` returns error if non-quosures are input", {
+  USUBJID <- "01-701-1015" # nolint
+
+  expect_error(
+    object = quo_c(quo(USUBJID), USUBJID)
+  )
+})
+
 # quo_not_missing ----
-## Test 1: `quo_not_missing` returns TRUE if no missing argument ----
-test_that("quo_not_missing Test 1: `quo_not_missing` returns TRUE if no missing argument", {
+## Test 3: `quo_not_missing` returns TRUE if no missing argument ----
+test_that("quo_not_missing Test 3: `quo_not_missing` returns TRUE if no missing argument", {
   test_fun <- function(x) {
     x <- rlang::enquo(x)
     assertthat::assert_that(quo_not_missing(x))
```

Comment:
I think these are pacakge imports and don't need `pkgname::`  only packages with suggests like tibble.  Can you remove please if you agree